docs/DeveloperGuide.md

```diff
@@ -504,3 +466,54 @@ testers are expected to do more *exploratory* testing.
 2. Dealing with corrupted data files 
    1. Add a new field `"newField" : "newField"` to a client. Expected: All data is lost. The app starts on a clean slate.
    2. Remove `"insurancePlans"` field from a client. Expected: All data is lost. The app starts on a clean slate.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix : Planned Enhancements**
+Do note we have 4 members in our group.
+
+### 1. addClaim ID
+A single claim ID can be added to multiple users. i.e. `B1100` can be added to both client `A` and client `B` with no error.
+This will be fixed in a future version such that claim ID is unique across all clients.
+
+### 2. addClaim amount
+Claim amount can currently exceed 1 million. In the future, a restriction will be placed on the claim amount such that if
+the claim amount is over 1 million, it will be rejected with an appropriate error message. This is in line with our
+restriction on claim amount (claim amount cannot exceed 1 million)
+
+### 3. Add command and Edit Command does not allow clients to have same names yet allows two people to have the same contact details.
+Currently, the way the system checks if a person is a duplicate is simply by checking if the person has the same
+name or not. However, in future versions, we are planning to check if all details are the same before flagging it as a
+duplicate. This is because 2 people can share numbers, address and emails (eg a parent and child) or 2 clients can have
+the same full name with different contact details but it will be unreasonably rare for clients to have the same name,
+number, email and addresses simultaneously.
+
+### 4. Some common names are not allowed to be entered
+Currently, our app does not allow some common Singaporean names with some special characters to be entered. An example
+is "s/o" or "d/o". This will be fixed in a future version because we will change the way we parse the add and edit
+commands.
+
+### 5. No Wrapping for very long ui details
```

Comment:
"Wrapping" should be all lower case here and "ui" should be uppercase

Reply:
good catch